.github/workflows/release.yml

```diff
@@ -13,6 +13,13 @@ jobs:
             - name: Checkout
               uses: actions/checkout@v3
 
+            - name: Publish to npm
+              run: |
+                npm config set //registry.npmjs.org/:_authToken '${NPM_TOKEN}'
+                npm publish
+              env :
+                NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
```

Comment:
No build?

Reply:
I forgot that publish doesn't run prepare.
I'll add the build step later today

Reply:
Done

Reply:
It does, but https://snyk.io/blog/github-actions-to-securely-publish-npm-packages/

Also, `prepare` requires `npm ci --ignore-scripts`.

Reply:
At this point it would almost be simpler to just `npm install`

Reply:
But yeah … npm ci etc is preferable, I'm not advocating for npm install … forget what I said

Reply:
Yeah, it explicity runs `backport` now

Reply:
@wojpawlik is still right that you'll need to install the deps first

Reply:
Hm... true

Reply:
Should be good now